ICRCs/ICRC-97/ICRC-97.md

```diff
@@ -0,0 +1,101 @@
+
+### Metadata properties
+
+Below table lists all (optional) metadata properties defined in this standard. Each property is defined in both ICRC-3
+and JSON type to support both on-chain `Value` and off-chain JSON metadata, respectively.
+
+| Metadata Property | Optional | ICRC-3 Type                             | JSON Type | Description                                             |
+|-------------------|----------|-----------------------------------------|-----------|---------------------------------------------------------|
+| external_url      | Yes      | variant { Text }                        | string    | URL that allows the user to view the item on your site. |
+| name              | Yes      | variant { Text }                        | string    | Plain text.                                             |
+| description       | Yes      | variant { Text }                        | string    | Markdown.                                               |
+| assets            | Yes      | variant { Array = vec variant { Map } } | object[]  | List of assets ordered by priority descending.          |
+| attributes        | Yes      | variant { Array = vec variant { Map } } | object[]  | List of attributes ordered by priority descending.      |
+
+The following table lists all (optional) asset properties.
+
+| Asset Property | Optional | ICRC-3 Type      | JSON Type       | Description                                                                    |
+|----------------|----------|------------------|-----------------|--------------------------------------------------------------------------------|
+| url            | No       | variant { Text } | string          | URL that returns the asset e.g. a PNG image (protocol is not limited to HTTP). |
+| mime           | No       | variant { Text } | string          | Mime type as defined in RFC 6838.                                              |
+| sha256_hash    | Yes      | variant { Blob } | string (base64) | SHA-256 hash of HTTP response body bytes returned from above url.              |
+| purpose        | Yes      | variant { Text } | string          | Indicate purpose of asset.                                                     |
+
+Below list of purpose values are part of the ICRC-97 standard, this list could be extended by other standards.
+Purpose values can define additional asset properties.
+
+| Purpose Value  | Description                                                                                                                                                 |
+|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| icrc97:image   | Original size image that is shown on e.g. item details page. Additional optional `width` and `height` properties define the dimensions in number of pixels. |
```

Comment:
can you provide an example on how this would be defined?

Reply:
I'll add examples to the spec